package.json

```diff
@@ -274,7 +273,8 @@
     "serve": "^11.0.0",
     "source-map-explorer": "^2.2.2",
     "svg2png": "^4.1.1",
-    "webpack-bundle-analyzer": "^3.6.0"
+    "webpack-bundle-analyzer": "^3.6.0",
+    "workbox-webpack-plugin": "^4.3.1"
```

Comment:
why workbox plugin was downgraded?

Reply:
@sirpy some properties when generating the service worker got deprecated in the version react-native had. I kept the version in master that we already know it works fine with the webpack config